src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -431,6 +447,14 @@ public GenInputsAbstract(
   public static boolean small_tests = false;
 
   /**
+   * Whether to use DigDog weighted sequences in sequence selection. Note that this weighting scheme
```

Comment:
"DigDog weighted sequences" is not defined.  It needs an explanation, either here or in the manual.  The manual needs to be self-contained, preferably without reference to external documents.